signup.js

```diff
@@ -0,0 +1,52 @@
+import { validateEmail, validatePassword, togglePasswordVisibility } from './validation.js';
+
+function isEmailInUse(email) {
+  return email === "test@codeit.com";
+}
+
+function validateConfirmPassword() {
+  const password = document.getElementById("password-input").value;
+  const confirmPassword = document.getElementById("confirm-password-input").value;
+  return password === confirmPassword ? "" : "비밀번호가 일치하지 않아요.";
+}
+
+document.addEventListener("DOMContentLoaded", function() {
```

Comment:
```suggestion
document.addEventListener("DOMContentLoaded", () => {
```

만약 익명함수를 사용할거라면 화살표 함수를 써보는건 어떨까요?
함수 선언문을 사용하면 this 바인딩 예측이 힘들어지고 함수 선언문을 기명으로 사용하면 사용처가 없어 기명으로한 의미가 없어지게 됩니다.
하지만 화살표 함수를 사용하게 되면 렉시컬 this로 동작해서 this 예측이 훨씬 쉬워지고 무조건 표현식으로만 사용해야하기 때문에 익명 함수로서 많이 사용되고 있어 권장드립니다.

Reply:
아하 감사합니다. 화살표 함수 쓰고싶은데 손에 익지 않아서 잘 쓰지 않게 되네요,, 
고쳐보도록 하겠습니다!